pkg/chain/key.go

```diff
@@ -1,10 +1,11 @@
-package eth
+package chain
 
 import (
 	"github.com/keep-network/keep-ecdsa/pkg/ecdsa"
 	"github.com/keep-network/keep-ecdsa/pkg/utils/byteutils"
 )
 
+// FIXME Why is this in the chain package? 🤔
```

Comment:
The serialization here is specific for Keep ECDSA contract, tBTC, and Ethereum, a mix of these three. The idea was that it is chain specific, but probably will be the same for other Ethereum-related chains. 

E.g. https://github.com/keep-network/keep-ecdsa/blob/4f172f1cedd89b794c679382f8dbdf40d4234bce/solidity/contracts/AbstractBondedECDSAKeep.sol#L594-L606

I'm fine with moving it somewhere else.

Reply:
Sounds like it might be a good fit for the new `ethlike` package or similar, perhaps? Def feels like something that should be pulled up to keep-common.